frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx

```diff
@@ -1,14 +1,30 @@
 import * as React from 'react';
-import { FormGroup, Checkbox } from '@patternfly/react-core';
+import { FormGroup, Checkbox, Radio } from '@patternfly/react-core';
+import { ResourceDropdown, getName } from '@console/shared';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { NetworkAttachmentDefinitionModel } from '@console/network-attachment-definition-plugin';
+import { NetworkAttachmentDefinitionKind } from '@console/network-attachment-definition-plugin/src/types/types';
+import { Firehose } from '@console/internal/components/utils';
+import { NetworkType } from '../types';
+import './_configure.scss';
+
+const resources = [
+  {
+    isList: true,
+    kind: referenceForModel(NetworkAttachmentDefinitionModel),
+    namespace: 'openshift-storage',
```

Comment:
If the NAD is in `openshift-storage` namespace, then it can't be used in other namespaces.
For example, an application using OCS as storage backend which is in namespace other than openshift-storage won't be able to utilize multus.

Only the NADs in the `default` namespace can be used accross namespaces. So we need to list NADs from the `default` namespace as well.
The NADs from the `default` namespace is referenced as `default/<name-of-NAD>` in the network spec
e.g., 
```yaml
public: "default/ocs-public-nw"
cluster: "default/ocs-cluster-nw"
```